.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,26 +9,26 @@
 
 <!--- Please provide a general summary of the Pull Request in the Title above -->
 
-### Description (*)
+**Description** (*)
 <!---
     Please provide a description of the changes proposed in the pull request.
     Letting us know what has changed and why it needed changing will help us validate this pull request.
 -->
 
-### Fixed Issues (if relevant)
+**Fixed Issues (if relevant)**
 <!---
     If relevant, please provide a list of fixed issues in the format magento/magento2-phpstorm-plugin#<issue_number>.
     There could be 1 or more issues linked here and it will help us find some more information about the reasoning behind this change.
 -->
 1. magento/magento2-phpstorm-plugin#<issue_number>: Issue title
 
-### Questions or comments
+**Questions or comments**
 <!---
 	If relevant, here you can ask questions or provide comments on your pull request for the reviewer
 	For example if you need assistance with writing tests or would like some feedback on one of your development ideas
 -->
 
-### Contribution checklist (*)
+**Contribution checklist** (*)
  - [ ] Pull request has a meaningful description of its purpose
  - [ ] All commits are accompanied by meaningful commit messages
 
```

Comment:
- [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

Reply:
Good catch